src/server/game/Entities/Player/Player.cpp

```diff
@@ -2216,6 +2219,8 @@ void Player::SetGMVisible(bool on)
         m_ExtraFlags |= PLAYER_EXTRA_GM_INVISIBLE;          //add flag
 
         SetAcceptWhispers(false);
+
+        SetAcceptMail(false);
```

Comment:
Seems kind of silly to be to block recieving mails when the GM uses the invisibility command.

Reply:
Mm maybe yes.

But then I'd argue the same for whisper. Perhaps both should be decided by the player and not when we go invis or not. Thoughts?

Reply:
@Kitzunu Whisper is fine since the GM goes invis and wishes to not be noticed by other players. With SetAcceptMail he can be noticed when they try to send him a mail and receive an error message